src/Microsoft.Kusto.ServiceLayer/LanguageServices/BindingQueue.cs

```diff
@@ -202,15 +202,6 @@ protected void RemoveBindingContext(string key)
                 {
                     // disconnect existing connection
                     var bindingContext = this.BindingContextMap[key];
-                    if (bindingContext.ServerConnection != null && bindingContext.ServerConnection.IsOpen)
```

Comment:
Removed ServerConnection from BindingContext so all it's references are deleted.